lib/config/rule-validator.js

```diff
@@ -35,16 +35,30 @@ function findRuleDefinition(ruleId, config) {
         pluginName = ruleIdParts.join("/");
     }
 
-    if (!config.plugins || !config.plugins[pluginName]) {
-        throw new TypeError(`Key "rules": Key "${ruleId}": Could not find plugin "${pluginName}".`);
```

Comment:
I think this was a better message for the case when plugin doesn't exist. Now it will be always `Could not find "bar" in plugin "foo".` regardless of whether plugin doesn't exist or rule doesn't exist in the plugin.

Reply:
Fair point. I'll change that.